src/assets/wise5/authoringTool/add-project/add-project.component.html

```diff
@@ -0,0 +1,52 @@
+<div id="top" class="view-content view-content--with-sidemenu">
+  <div class="l-constrained" fxLayout="column">
+    <div class="node-content md-whiteframe-1dp" style="position: relative">
+      <h5 i18n>Enter a title for the new unit</h5>
+      <form
+        role="form"
+        [formGroup]="addProjectFormGroup"
+        (keydown.enter)="create(); $event.preventDefault()"
+      >
+        <mat-form-field appearance="fill" class="title-field">
+          <mat-label i18n>Unit Title</mat-label>
+          <input
+            #titleField
+            autofocus
+            matInput
+            type="text"
+            id="title"
+            name="title"
+            formControlName="title"
+            (keyup)="$event.keyCode == 13 && create()"
+            required
+          />
+          <mat-error *ngIf="addProjectFormGroup.controls['title'].hasError('required')" i18n>
+            Unit Title is required
+          </mat-error>
+        </mat-form-field>
+        <mat-divider></mat-divider>
+        <div fxLayout="row">
+          <button
+            mat-button
+            class="mat-primary"
+            (click)="cancel(); $event.preventDefault()"
+            aria-label="cancel"
+            i18n
+          >
+            Cancel
+          </button>
+          <button
+            mat-button
+            class="mat-raised-button mat-primary"
```

Comment:
You can change this to

mat-raised-button
color="primary"

and remove the class="mat-raised-button mat-primary"